tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -636,20 +637,57 @@ def test_symdifference(self):
 
     def test_buffer(self):
         "Testing buffer()."
-        for bg in self.geometries.buffer_geoms:
+        bg = self.geometries.buffer_geoms[0]
+        g = fromstr(bg.wkt)
+
+        # Can't use a floating-point for the number of quadsegs.
+        with self.assertRaises(ctypes.ArgumentError):
+            g.buffer(bg.width, quadsegs=1.1)
+
+        self._base_test_buffer(self.geometries.buffer_geoms, 'buffer')
+
+    def test_buffer_with_style(self):
+        "Testing buffer_with_style()."
```

Comment:
I think this docstring can be removed. I understand that it was added to match `test_buffer`'s definition but it's not adding any value over the test method name.

Reply:
Done. 👍